metadata_generation.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
```

Comment:
Please rename this script to **combine_mimic_metadata.py** and move it to the "tools" directory

Reply:
Done